tests/contenttypes_tests/test_views.py

```diff
@@ -59,7 +92,7 @@ def test_shortcut_with_absolute_url_including_scheme(self):
             response = self.client.get(short_url)
             self.assertRedirects(response, obj.get_absolute_url(), fetch_redirect_response=False)
 
-    def test_shortcut_no_absolute_url(self):
```

Comment:
This seem to have been removed throughout time. Can't find these functions anywhere. Should I add them anyway? Moreover, the file seems to have been moved. Could someone confirm before I make any changes?